db/migrate/20171209171303_rename_events_to_event_instances.rb

```diff
@@ -0,0 +1,66 @@
+class RenameEventsToEventInstances < ActiveRecord::Migration[5.1]
+  # Stub models for migration in case they get renamed in the future
+  class MyEventInstance < ActiveRecord::Base
+    self.table_name = 'event_instances'
+  end
+  class MyEvent < ActiveRecord::Base
+    self.table_name = 'events'
+  end
+
+  def up
+    rename_event_instances_and_create_new_events_table
+
+    # Creating and naming nu-events isn't directly reversible
+    MyEventInstance.all.each do |event_instance|
+      ev = MyEvent.create!(name: event_instance.name)
+      event_instance.event_id = ev.id
+      event_instance.save
+    end
+
+    remove_event_instance_name_col
+  end
+
+  def down
+    revert { remove_event_instance_name_col }
```

Comment:
Ooh, didn't know you could do this. Interesting!

Reply:
super readable right?

Reply:
Ja!